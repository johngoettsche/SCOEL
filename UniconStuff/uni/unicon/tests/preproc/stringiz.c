/* test the strinizing operator */
#define f(a) >#a<
f()
f(1)
f(  1    2    3  )
f("hi")
f(1 "bye"  2)
f('\n')
f("\x01\n")
#begdef ml(x,y)
#x,
  #  y
#enddef
ml(name, +)
