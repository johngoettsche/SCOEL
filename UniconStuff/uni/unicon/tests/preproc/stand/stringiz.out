#line 2 "stringiz.c"

>""<
>"1"<
>"1 2 3"<
>"\"hi\""<
>"1 \"bye\" 2"<
>"'\\n'"<
>"\"\\x01\\n\""<

"name",
  "+"

